ServiceWebsite/ServiceWebsite/ClientApp/src/app/pages/unsupported-browser/unsupported-browser.component.ts

```diff
@@ -19,11 +19,11 @@ export class UnsupportedBrowserComponent implements OnInit {
     this.supportedBrowsers.push(new SupportedBrowserModel('Chrome', 'Chrome for Windows/Mac'));
     this.supportedBrowsers.push(new SupportedBrowserModel('Firefox', 'Firefox for Windows/Mac'));
     this.supportedBrowsers.push(new SupportedBrowserModel('Safari', 'Safari for Mac/iPad'));
-    this.supportedBrowsers.push(new SupportedBrowserModel('Edge Chromium', 'Edge Chromium for Windows'));
-    this.supportedBrowsers.push(new SupportedBrowserModel('Edge', 'Edge for Windows'));
+    this.supportedBrowsers.push(new SupportedBrowserModel('Edge Chromium', 'Edge for Windows'));
   }
 
   ngOnInit() {
+    // This is intentional
```

Comment:
Could remove the code smell

Reply:
The code smell is due the empty block, the //comments fixed it, suggested by sonarcloud